diagnostic.md

```diff
@@ -33,7 +42,9 @@ y = 10
 After each line of code executes, what are the values of x and y?  Do the variables change?  Do their values?
 
 <!-- solution below -->
-
+value of x is 4
+value of y is 3
+the variables and values change
```

Comment:
`x` and `y` are also both reassigned, so their values do change. Take a look at the code again